packages/devtools-components/src/tree.js

```diff
@@ -2,7 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const { DOM: dom, createClass, createFactory, PropTypes } = require("react");
+const { DOM: dom, createClass, createFactory, createElement, PropTypes } = require("react");
+const InlineSVG = require("svg-inline-react");
+const svgArrow = require("./images/arrow.svg");
+require("./tree.css");
```

Comment:
We made some changes to tree so that it would work in chrome, as well as support drag events that left the window. 

It'd be nice to look at the commits to this file and make sure that we address them.